guides/user/kubernetes-v2/deploy-helm/index.md

```diff
@@ -8,8 +8,7 @@ sidebar:
 {% include toc %}
 
 Spinnaker surfaces a "Bake (Manifest)" stage to turn templates into manifests
-with the help of a templating engine. Currently, the only supported templating
-engine is [Helm](https://helm.sh/), by relying on the `helm template` command.
+with the help of a templating engine. [Helm](https://helm.sh/) relies on the `helm template` command.
```

Comment:
Small nit, should we mention Kustomize here as well?

Reply:
I added a section to the end of each page linking to the other.